pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -96,37 +96,51 @@ func machinesEqual(m1 *clusterv1.Machine, m2 *clusterv1.Machine) bool {
 	return m1.Name == m2.Name && m1.Namespace == m2.Namespace
 }
 
+// isNodeJoin determines if a machine, in scope, should join of the cluster.
 func (a *Actuator) isNodeJoin(scope *actuators.MachineScope, controlPlaneMachines []*clusterv1.Machine) (bool, error) {
 	switch set := scope.Machine.ObjectMeta.Labels["set"]; set {
 	case "node":
+		// non control plane machines will always join the cluster.
 		return true, nil
 	case "controlplane":
+		// control plane machines will join the cluster if the cluster has an existing control plane.
```

Comment:
```suggestion
		// Control plane machines will join the cluster if the cluster has an existing control plane.
```

Reply:
Fixed. Also wanted to fix `Control plane` to `Controlplane`.